### ISSUE TITLE
AngularJS Compatibility?

### ISSUE THREAD

**Issue by solarisfire** [gfranko/jquery.tocify.js#89]
Would love to have this working with AngularJS.

I have jquery, jquery-ui, and tocify included, and if I call .tocify() on a div after the AngularJS digest cycle using $timeout it gives me a table of contents.

However clicking on any element in the toc just directs me to a different route, not to the right position on the page.

**Comment by agez** [comment]
I think that you have to change "history" to false. It is true by default and creates hashes in the URL. 

We also use tocify within an AngularJs app for about 3 years. And as far as I remember we had the same issues with routing. Switching "history" to false did the trick.

HTH.

**Comment by solarisfire** [comment]
Yup that worked, however it won't stick to the top of the page on scroll. Guess I could always supplement it with something like ngSticky for that...

**Comment by solarisfire** [comment]
Getting MAJOR performance issues though, running tocify on the content after each route change. But looks like tocify isn't cleaning up after changing route so is leaking memory...
